src/Evolution/Systems/Cce/WorldtubeDataManager.hpp

```diff
@@ -49,9 +49,16 @@ void populate_hypersurface_boundary_data(
     size_t l_max, size_t buffer_depth);
 }  // namespace detail
 
+namespace Tags {
+using klein_gordon_worldtube_boundary_tags =
+    tmpl::list<Tags::BoundaryValue<Tags::KleinGordonPsi>,
+               Tags::BoundaryValue<Tags::KleinGordonPi>>;
+} // namespace Tags
```

Comment:
Should these tags go somewhere else? Where are the tags for the other two WorldtubeDataManagers kept?

Reply:
They are kept in `src/Evolution/Systems/Cce/BoundaryData.hpp`,  where the tags are computed from spacetime metric, etc. It's not relevant here. The only other place I could think of is to move them to `src/Evolution/Systems/Cce/Tags.hpp`.

Reply:
The `characteristic_worldtube_boundary_tags` list in `BoundaryData.hpp` seems to be basically the same as this `klein_gordon_worldtube_boundary_tags` list, just with more (and different) tags and potentially a different prefix. So I believe it makes sense to keep these two lists together so we don't have to go looking in two different places for a boundary tags list.

Also, having the list in a more "central" place allows us to reuse the type alias if we need (no idea if that's the plan or not).